src/node_crypto.cc

```diff
@@ -3069,7 +3069,9 @@ void CipherBase::Update(const FunctionCallbackInfo<Value>& args) {
   // Only copy the data if we have to, because it's a string
   if (args[0]->IsString()) {
     StringBytes::InlineDecoder decoder;
-    if (!decoder.Decode(env, args[0].As<String>(), args[1], UTF8))
+    bool result;
+    if (!decoder.Decode(env, args[0].As<String>(), args[1], UTF8).To(&result) ||
+        !result)
```

Comment:
tiny nit: `if (!decoder.Decode(…).FromMaybe(false))` also does the trick and is a bit more idiomatic (here and below)